staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -230,6 +230,7 @@ type Cacher struct {
 	// See: https://golang.org/pkg/sync/atomic/ for more information
 	incomingHWM storage.HighWaterMark
 	// Incoming events that should be dispatched to watchers.
+	// The number of incoming watchevents are currently 100(refer line 349)
```

Comment:
Not sure why this is needed.
Also, refrain from including line numbers in code comments since they can change often.

Reply:
Oh yes, my bad, gonna remove that.
As the issue mentioned, the word "incoming" is suggestive, but isn't obvious, so I thought that maybe it's missing the length of watchEvents.

Reply:
If I understand correctly, the issue advocates for further clarity around `incomingHWM` and not the `incoming` channel.

Reply:
Maybe adding documentation around what the `incoming` channel is and its intent might help increase clarity if its not immediately obvious right now when read.